File: NotePD/Service/FileService.cs
using System.IO;
using System.Windows.Forms;
using System;

namespace NotePD.Services
{
    public static class FileService
    {
        public static Tuple<string, string> Open()
        {
            OpenFileDialog dlg = new OpenFileDialog();

            if (dlg.ShowDialog() == DialogResult.OK)
            {
                string text = File.ReadAllText(dlg.FileName);
                return Tuple.Create(dlg.FileName, text);
            }

            return null;
        }

        public static void Save(string path, string content)
        {
            File.WriteAllText(path, content);
        }

        public static string SaveAs(string content)
        {
            SaveFileDialog dlg = new SaveFileDialog();

            if (dlg.ShowDialog() == DialogResult.OK)
            {
                File.WriteAllText(dlg.FileName, content);
                return dlg.FileName;
            }

            return null;
        }
    }
}

File: JoaquimPadPy/file_system/file_service.py
class FileService:

    def open_file(self, path):
        with open(path, "r", encoding="utf-8") as f:
            return f.read()

    def save_file(self, path, content):
        with open(path, "w", encoding="utf-8") as f:
            f.write(content)
